lib/src/components/NxTextInput/NxTextInput.scss

```diff
@@ -37,4 +37,6 @@
   font-size: $nx-font-size-xs;
   line-height: 16px;
   margin-top: $nx-spacing-xs;
+  position: absolute;
+  bottom: -20px;
```

Comment:
This is both slightly wrong and unneeded.  If you use `position: absolute` without any `top`, `right`, `bottom`, or `left`, it lays the element out at the location it would have been statically.

Reply:
> This is both slightly wrong and unneeded. If you use `position: absolute` without any `top`, `right`, `bottom`, or `left`, it lays the element out at the location it would have been statically.

https://github.com/sonatype/sonatype-react-shared-components/pull/180/commits/e1c847b3a8fb8f8e968c7f8331f087ca8a3754af